source/worksheet/worksheet.cpp

```diff
@@ -1331,4 +1331,25 @@ bool worksheet::is_empty() const
     return d_->cell_map_.empty();
 }
 
+int worksheet::zoom_scale() const
+{
+    if (!has_view())
+    {
+        return 100;
```

Comment:
This line is not covered: see [coverage report](https://xlnt-community.github.io/xlnt-coverage/0f94823445aa3ae028de50ae6abd1a1086dabaa8/review/index.html)